### ISSUE TITLE
[BE] Create a simple data storage

### ISSUE THREAD

**Issue by AnnaH3003** [amosproj/amos2023ws01-ticket-chat-ai#34]
Prerequisite: #29 

First idea: Google Drive to save the initial input as well as the generated ticket. 

## Acceptance criteria
- [ ] input and generated tickets are saved
- [ ] input and generated tickets can be viewed

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by M-HRL** [comment]
@WebFa98, do you want to review my changes in the end or do you want to pair on this work?

**Comment by WebFa98** [comment]
i dont know if there is much to pair on this work, i can just review it.

**Comment by M-HRL** [comment]
I have already added some code in the feature branch https://github.com/amosproj/amos2023ws01-ticket-chat-ai/tree/feature/34-add-database-to-backend

**Comment by M-HRL** [comment]
@WebFa98, I added a draft pull-rq that contains the app code, test code and some other changes. I would ask you to install mongodb com server on your local machine via the link in the README and install the added requirements again. After that, you should be able to execute the added test as described in the README if you're not on Windows, as the config for this os is missing. In this case, you could add the Windows start and stop routine for the db server.

**Comment by WebFa98** [comment]
currently the test fails for me at the second assertion where it determines the number of tickets in the collection

**Comment by M-HRL** [comment]
@WebFa98, you're right, it didn't work on my machine as well, if I deleted my database. That's because assert <var> means that it shouldn't be 0, but I wanted a check for not None. I fixed it.

**Comment by M-HRL** [comment]
But do you use Windows and did you add the necessary command in the database_routine.py? Otherwise, I will do it later this weekend.

**Comment by WebFa98** [comment]
👍The test passes now, i used Linux.
